wolfcrypt/src/port/Renesas/renesas_tsip_util.c

```diff
@@ -87,8 +191,7 @@ int tsip_hw_lock()
         /* this should not happens */
         return -1;
     }
-
-    WOLFSSL_MSG("leave tsip_sha_try_hw_lock");
+    /*WOLFSSL_MSG("leave tsip_sha_try_hw_lock");*/
```

Comment:
Looks like this message line should be removed or uncommented?

Reply:
They will be removed.